docs/source/models/gbd2017/causes/cause_model_template.rst

```diff
@@ -91,125 +82,193 @@ Disease Description
+  model.
+
+.. admonition:: Todo for template development
+
+  What else goes in the `Vivarium Modeling Strategy` section? Should there be
+  any other subsections besides `Scope` and `Assumptions and Limitations`?
 
 Cause Model Diagram
 -------------------
 
-State and Transition Data Tables
---------------------------------
+Data Description
+----------------
 
 .. admonition:: Todo for template development
 
-    - Decide on section heading names and structure. E.g. should this section be
-      called "Data Description" instead of "State and Transition Data Tables,"
-      like it was originally?
+  Decide on section heading name(s) and structure for the data section. See
+  comments by @Beatrix and @Nathaniel in `Pull Request 64`_ discussing the
+  following questions:
 
-      In `Pull Request 91`_, @James said:
+  - Should this section be called "Data Description" as it is now, or would
+    something like "State and Transition Data Tables" be more descriptive?
 
-        I think `Data Description` is fine.
+    In `Pull Request 91`_, @James said:
+
+      I think `Data Description` is fine.
 
-    - Is there anything else that belongs in a "Data Description" section
-      besides the data tables?
+  - Should this section have subsections (such as "State and Transition Data
+    Tables")? Is there anything else that belongs in a "Data Description"
+    section besides the data tables?
 
-      In `Pull Request 91`_, @Yongquan said:
+    In `Pull Request 91`_, @Yongquan said:
 
-        one data-relevant section needed for more complicated models is
-        non-standard data sources where external calculation occurs.
+      one data-relevant section needed for more complicated models is
+      non-standard data sources where external calculation occurs.
 
-    - Add instructions in other sections, including:
+    Does "non-standard data sources" warrant a subsection, even if we don't need
+    it for all cause models?
 
-      - Using editable :file:`.svg` format for figures
-      - For cause model diagram: description of what the bubbles and arrows
-        represent. Maybe include svg templates for common diagrams like SI, SIS,
-        SIR, etc.
-      - For cause hierarchy: description of our strategy for making cause
-        hierarchy diagrams (rules + example)
-      - For data tables: Template tables and instructions for filling them in
+Validation Criteria
+-------------------
 
-Model Assumptions and Limitations
----------------------------------
+References
+----------
 
 .. admonition:: Todo for template development
 
-  - Are the sections in a good order? In `Pull Request 91`_, @Lu said:
+  Decide on section names and overall structure.
 
-      The template looks good to me. I was putting the model assumptions and
-      limitations section right after the cause model diagram. But I think
-      this order makes more sense.
+  **Question:** Are the sections in a good order?
```

Comment:
I like the section order as stands in this PR

Reply:
I'm also happy with how these are ordered. I think the one drawback is that the general disease description is going to be somewhat tailored towards what the GBD and vivarium models are concerned with, but that doesn't necessarily become evident until you've read the "restrictions" and "scope" sections below.

But I think the benefit of keeping this section first --in that it is quite helpful in orienting the reader-- outweighs the above concern.